docs/experimental.md

```diff
@@ -4,10 +4,8 @@ The following features are experimental and subject to change:
 
 - [Windows containers](https://github.com/containerd/nerdctl/issues/28)
 - [FreeBSD containers](./freebsd.md)
-- Importing an external eStargz record JSON file with `nerdctl image convert --estargz-record-in=FILE` .
+- Flags of `nerdctl image convert`: `--estargz-record-in=FILE` and `--zstdchunked-record-in=FILE` (Importing an external eStargz record JSON file), `--estargz-external-toc` (Separating TOC JSON to another image).
```

Comment:
Experimental flags need to check the `experimental` config

https://github.com/containerd/nerdctl/blob/b71443b2a3abc6288944c72229a2e21a1ae7ec6e/cmd/nerdctl/pull.go#L157-L164

Reply:
Fixed